power_grid_model_c/power_grid_model/include/power_grid_model/main_model.hpp

```diff
@@ -545,9 +568,10 @@ class MainModelImpl<ExtraRetrievableTypes<ExtraRetrievableType...>, ComponentLis
                 }
```

Comment:
we can remove this part as it is the whole intention here to avoid copy the whole model.

```
                if (!independent) {
                    model = base_model;
                }
```

Reply:
right, i did this and then i didn't due to the investigation in my comment above 😬 will fix